### ISSUE TITLE
Improvement: New dfeshiny functionality - custom_disconnect_message and cookies_banner_server

### ISSUE THREAD

**Issue by JT-39** [dfe-analytical-services/local-authority-interactive-tool#69]
**Improvement Overview**

`{dfeshiny}` has had an update and a few functions have some additional functionality which are of use.

**Current Issue**

Currently I have a temporary function `cookies_banner_server_jt()` which allows the specification of the navigation panel id (in my case this is `"pages"`). The `dfeshiny::cookies_banner_server()` was strict and did not permit for this `id` to be different to the default `"navlistPanel"`. However, now this has been relaxed. So the `dfeshiny::cookies_banner_server()` can be used.

Also, the `dfeshiny::custom_disconnect_message()` has been updated to allow for extra email addresses to be included (for example Darlington.BRIDGE@education.gov.uk) and an extra link to be available for refresh (for example, the default https://department-for-education.shinyapps.io/local-authority-interactive-tool/ - which has no bookmarking).

**Proposed Enhancement**

- [ ] Remove the `cookies_banner_server_jt()` function, and update the `{dfeshiny}` package to use the new `dfeshiny::cookies_banner_server()`.
- [ ] Utilise the new functionality of the `dfeshiny::custom_disconnect_message()` to point towards Darlington.BRIDGE@education.gov.uk and an additional link to https://department-for-education.shinyapps.io/local-authority-interactive-tool/ .

**Impact**

Tidier code and inline with the `{dfeshiny}` package. Better disconnect button.

**Additional Information**

Currently waiting on a couple changes to `{dfeshiny}` to do these updates:

- https://github.com/dfe-analytical-services/dfeshiny/issues/75
- https://github.com/dfe-analytical-services/dfeshiny/issues/76

**Comment by JT-39** [comment]
Still waiting on `{dfeshiny}` to add in the option to have both a refresh link to the current bookmarked site and also refresh the site back to its default state:

https://github.com/dfe-analytical-services/dfeshiny/issues/80

So, will create a separate issue for this as the removing of the `cookie_banner_sever_jt()` function has been complete after the typo correction in `dfeshiny::cookie_banner_server()`.

**Comment by JT-39** [comment]
This is the new issue for the custom disconnect message:

https://github.com/dfe-analytical-services/local-authority-interactive-tool/issues/78
